### PR TITLE
Fix some -Wfloat-conversion warnings

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

**Review by mbkma — APPROVED** [review]
LGTM

File: src/dlg-new.c [hunk 1]
```diff
@@ -292,7 +293,9 @@ dlg_new_archive (FrWindow  *window,
         g_object_unref (settings);
 
         settings = g_settings_new (ENGRAMPA_SCHEMA_BATCH_ADD);
-        gtk_spin_button_set_value (GTK_SPIN_BUTTON (data->n_volume_spinbutton), g_settings_get_int (settings, PREF_BATCH_ADD_VOLUME_SIZE) / MEGABYTE);
+	size = g_settings_get_int (settings, PREF_BATCH_ADD_VOLUME_SIZE);
```
Comment (raveit65) [inline src/dlg-new.c:296]: Indent

**Review by raveit65 — APPROVED** [review]
Beside the little code-style issue it looks good.

File: src/dlg-new.c (hunk 1 above)
Comment (rbuj) [reply src/dlg-new.c]: done

**Pull request merged**